src/docs/code-review.mdx

```diff
@@ -204,7 +204,7 @@ Please be pragmatic, and consider the cost of each incremental request for chang
 - We recommend checking for open code reviews at the start and end of every day.
 - [Github's Review Requests tab](https://github.com/pulls/review-requested) can be a helpful place to keep track of these.
 
-### Examples of *pretty good* code reviews
```

Comment:
it was meant to be examples 😭 - but until we add more let's keep the correction!

Reply:
Hopefully there's another good example some day! :P <3